packages/core/http/core-http-resources-server-mocks/BUILD.bazel

```diff
@@ -0,0 +1,110 @@
+load("@npm//@bazel/typescript:index.bzl", "ts_config")
+load("@build_bazel_rules_nodejs//:index.bzl", "js_library")
+load("//src/dev/bazel:index.bzl", "jsts_transpiler", "pkg_npm", "pkg_npm_types", "ts_project")
+
+PKG_DIRNAME = "core-http-resources-server-mocks"
+PKG_REQUIRE_NAME = "@kbn/core-http-resources-server-mocks"
+
+SOURCE_FILES = glob(
+  [
+    "**/*.ts",
+  ],
+  exclude = [
+    "**/*.config.js",
+    "**/*.test.*",
+    "**/*.stories.*",
+    "**/__snapshots__/**",
+    "**/integration_tests/**",
+    "**/mocks/**",
```

Comment:
TIL the hard way to double-check what type of files we're excluding from the source.

Reply:
Yea 😅 mock packages must remove the `*.mock.*` line from here...